modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyDetail/index.tsx

```diff
@@ -14,19 +15,52 @@
 
 import './index.scss';
 
-import {useState} from 'react';
+import {useEffect, useState} from 'react';
 
 import NavigationBar from '../../../../../common/components/navigation-bar';
 import {getWebDavUrl} from '../../../../../common/utils/webdav';
 
-const PolicyDetail = () => {
+type PolicyDetails = {
+	annualMileage: number;
+	creditRating: string;
+	features: string;
+	firstName: string;
+	gender: string;
+	highestEducation: string;
+	maritalStatus: string;
+	model: string;
+	occupation: string;
+	ownership: string;
+	primaryUsage: string;
+};
+
+const PolicyDetail = ({application}: any) => {
 	enum navBarLabels {
 		Vehicles = 'Vehicles',
 		Drives = 'Drives',
 	}
-	const amountOfPolicyDetail = [0];
 	const navbarLabel = [navBarLabels.Vehicles, navBarLabels.Drives];
 	const [active, setActive] = useState(navbarLabel[0]);
+	const [applicationData, setApplicationData] = useState<any>();
```

Comment:
Please, advise the correct type for it.
```suggestion
	const [applicationData, setApplicationData] = useState<???>();
```

Reply:
All ApplicationData attributes are typed in PolicyDetailsType and ApplicationPolicyDetailsType.